src/mudlet-mapper.xml

```diff
@@ -3549,6 +4231,112 @@ end</script>
                 <packageName></packageName>
                 <regex>^room marks$</regex>
```

Comment:
Um, I am giving up on looking through this, just too much to digest, I _hope_ it all works, please indicate whether you'd prefer me to keep going... :disappointed: